### DISCUSSION TITLE
404 Not Found https://packages.sury.org/php focal Release | Need help with php installation

### DISCUSSION THREAD

**Discussion by Fyouzard** [oerdnj/deb.sury.org#2065] [Q&A] [answered]
Hey ! I'm trying to install Azuriom CMS on a VPS with Ubuntu Server 20.04. While [installing the CMS](https://azuriom.com/fr/docs/installation) I need to run this command :

`wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg
echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/php.list`

At this point nothing gets wrong but when I do `apt update -y` I have this error : 

> Hit:1 http://asi-fs-d.contabo.net/ubuntu focal InRelease
Hit:2 http://asi-fs-d.contabo.net/ubuntu focal-updates InRelease
Hit:3 http://asi-fs-d.contabo.net/ubuntu focal-backports InRelease
Hit:4 http://ppa.launchpad.net/ondrej/apache2/ubuntu focal InRelease
Hit:5 http://ppa.launchpad.net/ondrej/php/ubuntu focal InRelease
Ign:6 https://packages.sury.org/php focal InRelease
Err:7 https://packages.sury.org/php focal Release
  404  Not Found [IP: 169.150.247.39 443]
Hit:8 http://security.ubuntu.com/ubuntu focal-security InRelease
Reading package lists... Done
E: The repository 'https://packages.sury.org/php focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

Anyone experience the same thing before ? Please help me !

**Comment by sleemanj** [comment, marked as answer]
packages.sury.org is Debain, you are using Ubuntu,

See FAQ

https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions#how-to-enable-the-debsuryorg-repository
